Brisk/Extensions/NSStatusItem+Extension.swift

```diff
@@ -2,7 +2,7 @@ import AppKit
 
 extension NSStatusItem {
     static func create(image: NSImage, menu: NSMenu) -> NSStatusItem {
-        let statusItem = NSStatusBar.system().statusItem(withLength: NSVariableStatusItemLength)
+        let statusItem = NSStatusBar.system.statusItem(withLength: NSStatusItem.variableLength)
```

Comment:
Can you omit the enum or is this just a constant?

Reply:
.variableLength seems to be a CGFloat, so NSStatusItem can't be omitted